src/ia-sharing-options.js

```diff
@@ -33,6 +34,7 @@ export class IASharingOptions extends LitElement {
       identifier: { type: String },
       sharingOptions: { type: Array },
       type: { type: String },
+      renderHeader: { type: Boolean },
```

Comment:
I don't recall the use case for disabling the rendering of the header for submenu items. Is that how we're deferring header rendering to the parent menu slider?

Reply:
Isa confirmed it's to allow the parent menu slider to render the header instead.